src/app/services/baw-api/baw-api.service.ts

```diff
@@ -223,11 +235,11 @@ export class BawApiService<
     if (body?.hasFormDataOnlyAttributes({ update: true })) {
       const formData = body.getFormDataOnlyAttributes({ update: true });
       return request.pipe(
-        mergeMap(() => this.httpPut(path, formData, multiPartHeaders)),
+        mergeMap(() => this.httpPut(path, formData, multiPartApiHeaders)),
         map(this.handleSingleResponse(classBuilder))
       );
     }
-    return request;
+    return request.pipe(tap(() => this.manager.delete(path)));
```

Comment:
should probably use their KeySerializer  https://github.com/ngneat/cashew#hack-the-library

![image](https://user-images.githubusercontent.com/1452968/171760514-33d05cc9-754a-4e0e-a8bc-119aa3726d17.png)


I know we aren't using any QSP for our get requests but if we do this would break